content/documentation/pages/1-installation/3-kubernetes/2-kubectl.md

```diff
@@ -0,0 +1,328 @@
+Docker image in the container specification, as the following example
+shows:
+
+        spec:
+          containers:
+          - name: skipper
+            image: springcloud/spring-cloud-skipper-server:{skipper-version}   #
+
+- You may change the version as you like.
+
+[tip | Multiple platform support]
+| Skipper includes the concept of
+| [platforms](https://docs.spring.io/spring-cloud-skipper/docs/current/reference/htmlsingle/#using-platforms),
+| so it is important to define the "`accounts`" based on the project
+| preferences.
+
+To use RabbitMQ as the messaging middleware, run the following command:
+
+    kubectl create -f src/kubernetes/skipper/skipper-config-rabbit.yaml
+
+To use Apache Kafka as the messaging middleware, run the following
+command:
+
+    kubectl create -f src/kubernetes/skipper/skipper-config-kafka.yaml
+
+Additionally, to use the [Apache Kafka Streams
+Binder](https://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/#_apache_kafka_streams_binder),
+update the `environmentVariables` attribute to include the Kafka Streams
+Binder configuraton in
+`src/kubernetes/skipper/skipper-config-kafka.yaml` as follows:
+
+    environmentVariables: 'SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT},SPRING_CLOUD_STREAM_KAFKA_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT}, SPRING_CLOUD_STREAM_KAFKA_STREAMS_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT},SPRING_CLOUD_STREAM_KAFKA_STREAMS_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT}'
+
+Run the following commands to start Skipper as the companion server for
+Spring Cloud Data Flow:
+
+    kubectl create -f src/kubernetes/skipper/skipper-deployment.yaml
+    kubectl create -f src/kubernetes/skipper/skipper-svc.yaml
+
+You can use `kubectl get all -l app=skipper` to verify that the
+deployment, pod, and service resources are running. You can use
+`kubectl delete all,cm -l app=skipper` to clean up afterwards.
+
+### Deploy Data Flow Server
+
+The deployment is defined in the
+`src/kubernetes/server/server-deployment.yaml` file. To control which
+version of Spring Cloud Data Flow server gets deployed, modify the tag
+used for the Docker image in the container specification, as follows:
+
+        spec:
+          containers:
+          - name: scdf-server
+            image: springcloud/spring-cloud-dataflow-server:%dataflow-version%      #
+
+You must specify the version of Spring Cloud Data Flow server that you want to deploy.
+
+1. Change the version as you like. This document is based on the
+   `%dataflow-version%` release. You can use the docker `latest` tag for
+   `BUILD-SNAPSHOT` releases.
+
+2. The Skipper service should be running and the `SPRING_CLOUD_SKIPPER_CLIENT_SERVER_URI` property in `src/kubernetes/server/server-deployment.yaml` should point to it.
+
+The Data Flow Server uses the [Fabric8 Java client
```

Comment:
Probably don't need to go into this level of detail.   i.e. fabric8

Reply:
I think we want to directly refer to the section - https://github.com/fabric8io/kubernetes-client#configuring-the-client so that users understand their options for configuration.